packages/app-cli/app/command-mv.js

```diff
@@ -7,25 +7,45 @@ const Note = require('@joplin/lib/models/Note').default;
 
 class Command extends BaseCommand {
 	usage() {
-		return 'mv <note> [notebook]';
+		return 'mv <item> [notebook]';
 	}
 
 	description() {
-		return _('Moves the notes matching <note> to [notebook].');
+		return _('Moves the given <item> (notes matching pattern in current notebook or one notebook) to [notebook]. If <item> is subnotebook and [notebook] is "root", will make <item> parent notebook');
 	}
 
 	async action(args) {
-		const pattern = args['note'];
+		const pattern = args['item'];
 		const destination = args['notebook'];
+		let folder = null;
 
-		const folder = await Folder.loadByField('title', destination);
-		if (!folder) throw new Error(_('Cannot find "%s".', destination));
+		if (destination != 'root') {
```

Comment:
```suggestion
		if (destination !== 'root') {
```
(This suggestion is based on the (still-under-review) [style guide](https://github.com/laurent22/joplin/pull/6657/files#diff-9139b1fb1702097e5361818684f9ed7b3a7c73ecc5dfc4105ba58e9e810b8a01R83-R92).)